lambdas/submit_job/function.py

```diff
@@ -0,0 +1,64 @@
+from botocore.exceptions import ClientError
+from summary_analysis_batch.utils import slack_webhook, s3_directory_exists
+from summary_analysis_batch.summary_batch_job import get_summary_analysis_step, submit_summary_batch_job
+import datetime
+
+RESULT_BUCKET = "gfw-pipelines-dev"
+RESULT_PREFIX = "geotrellis/results/{date_version}/{name}"
+RESULT_PREFIX = "geotrellis/results/{date_version}/{name}"
+RESULT_PATH = "s3://{}/{}"
+
+
+#  s3://gfw-pipelines-dev/geotrellis/results/v20191119/new_area_test/gladalerts/gladAlerts_20191119_2106/geostore/daily_alerts/part-00000-41e4a658-6254-48ba-a35c-46ea3297cc50-c000.csv
+
+def handler(event, context):
+    env = event["env"]
+    name = event["name"]
+    feature_src = event["feature_src"]
+    feature_type = event["feature_type"]
```

Comment:
You will also need to add a flag for glad or tcl (`--glad` or `--tcl`). Otherwise geotrellis with fail when extent falls outside glad/ annual loss coverage. In the check_new_area function I make sure, that these attributes are added to the geostore TSV